Move aws constants in dedicated YAML file.

La0 opened the pull request:
This file will be used by the [fuzzing team decision task](https://github.com/MozillaSecurity/fuzzing-tc) to build worker pools

I checked that this PR did *not* change anything when running `tc-admin diff` in comparison to current's master state.

La0 commented on generate/workers.py:
```diff
@@ -227,50 +228,12 @@ def aws(
-        },
-        "us-east-2": {
-            "no-inbound": "sg-00a9d64b3595c5088",
-            "docker-worker": "sg-0388de36e2f30ced2  u",
```
I removed the trailing `u` here as it looks like a typo.

petemoore replied:
I checked, and `sg-0388de36e2f30ced2` does exist in us-east-2, so that is correct now.

La0 commented:
I moved the constants in `config/aws.yml` and added a description at the top of the file.

We are just starting to build the fuzzing decision task, so it should support edition and removal of values, but i left a comment about reaching out to the fuzzing team anyway...

djmitche commented:
Oops, yamllint failed.  Fix that up and this is good to merge.  Thanks!
